unit_test/test_spdm_requester/get_csr.c

```diff
@@ -322,6 +332,9 @@ void libspdm_test_requester_get_csr_case1(void **state)
     /*init req_info*/
     libspdm_gen_req_info();
 
+    /*ensure that cached.csr exists in test_csr at the beginning*/
+    libspdm_clear_cached_csr();

```

Comment:
Probably best to call this from `libspdm_requester_get_csr_test_main`.

Reply:
Thanks. I have moved the code location.